### ISSUE TITLE
For some image preview is missing

### ISSUE THREAD

**Issue by byman64** [kisstkondoros/gutter-preview#9]
https://prnt.sc/gat4y7

`<figure>
        <h3>KEEPASS</h3>
        <img src="https://upload.wikimedia.org/wikipedia/commons/0/04/KeePass_icon.svg" alt="KeePass">
        <figcaption>
            <a href="http://keepass.info/" target="_blank" rel="noopener noreferrer" class="btn btn-default">keepass.info</a>
            <p>Free, open source, light-weight and easy-to-use password manager.</p>
        </figcaption>
    </figure>`

<pre>
<figure>
        <h3>KEEPASS</h3>
        <img src="https://upload.wikimedia.org/wikipedia/commons/0/04/KeePass_icon.svg" alt="KeePass">
        <figcaption>
            <a href="http://keepass.info/" target="_blank" rel="noopener noreferrer" class="btn btn-default">keepass.info</a>
            <p>Free, open source, light-weight and easy-to-use password manager.</p>
        </figcaption>
    </figure>
</pre>

**Comment by kisstkondoros** [comment]
It seems this is related to https://github.com/Microsoft/vscode/issues/30879

TLDR;
They've improved security by restricting the type and source of embedded content (e.g. protocol http, content type svg). By default this is in "strict mode" and it can not be turned off (AFAIK)

Unfortunately I can't fix this on my own, You may report this issue at https://github.com/Microsoft/vscode/

**Comment by kisstkondoros** [comment]
Partial fix provided in v0.10.2
